Behavioral.Automation.DemoBindings/Bootstrapper.cs

```diff
@@ -20,11 +26,61 @@ public Bootstrapper(IObjectContainer objectContainer, ITestRunner runner, Browse
             _browserRunner = browserRunner;
             _servicesBuilder = new DemoTestServicesBuilder(objectContainer, new TestServicesBuilder(_objectContainer));
         }
+        
+        private static void SetUpTestApp()
+        {
+            try
```

Comment:
`try ... catch` block is designed to catch exceptions, i.e. abnormal program behavior. Let's try and rewrite this